scripts/buildAmici.sh

```diff
@@ -11,15 +11,15 @@ mkdir -p ${AMICI_PATH}/build
 cd ${AMICI_PATH}/build
 CPPUTEST_BUILD_DIR=${AMICI_PATH}/ThirdParty/cpputest-master/build/
 CppUTest_DIR=${CPPUTEST_BUILD_DIR} cmake -DCMAKE_BUILD_TYPE=Debug ..
-make
+make VERBOSE=1
```

Comment:
remove verbose

Reply:
Oops. That came too late. Please remove in the open PR.